src/SimpleCalculator/obj/Debug/net6.0/SimpleCalculator.GeneratedMSBuildEditorConfig.editorconfig

```diff
@@ -7,4 +7,4 @@ build_property.InvariantGlobalization =
 build_property.PlatformNeutralAssembly = 

```

Comment:
any files inside of `obj` folder shouldn't be a part of github source code. Please use gitignore file